### ISSUE TITLE
Out of bounds error in trial segmented data stroke detection 

### ISSUE THREAD

**Issue by isolver** [isolver/OpenHandWrite#109]
Opening these test data exp_template_data.hdf5 in MarkWrite and changing the trial to DV_GO_Start / End gives the error in the screenshot. This doesn’t happen with DV_INST_.

![unnamed](https://cloud.githubusercontent.com/assets/1795082/11899023/537f26e2-a56a-11e5-877b-408e357c9a57.png)

**Comment by isolver** [comment]
Just tested this with the latest code base and I do not get an error when selecting DV_GO_START and DV_GO_END. 4 trials get loaded and I can selected each trial without error.

So my guess is that whatever bug existed a couple days ago has been fixed by some other changes I made. Bonus. ;)

I'm going to send an updated distro tonight (Monday night). Let me know if you run into any issues with it. I'll leave this issue open and only close it Wednesday if I do not hear from you.
